src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/AccessToken/OidcTokenHandlerFactory.php

```diff
@@ -92,35 +92,12 @@ public function addConfiguration(NodeBuilder $node): void
             ->arrayNode($this->getKey())
                 ->fixXmlConfig($this->getKey())
                 ->validate()
-                    ->ifTrue(static fn ($v) => !isset($v['algorithm']) && !isset($v['algorithms']))
-                    ->thenInvalid('You must set either "algorithm" or "algorithms".')
+                    ->ifTrue(static fn ($v) => !isset($v['algorithms']))
+                    ->thenInvalid('You must set "algorithms".')
```

Comment:
I think this can be removed, right?

Reply:
I think so yes
you might want to verify on a proto app of course